projects/ui/table/column-def.ts

```diff
@@ -0,0 +1,22 @@
+import { CdkColumnDef } from '@angular/cdk/table'
+import { Directive, Input } from '@angular/core'
+
+@Directive({
+  standalone: true,
+  selector: '[ckColumnDef]',
+  providers: [
+    { provide: CdkColumnDef, useExisting: CkColumnDef },
+    // {provide: 'CK_SORT_HEADER_COLUMN_DEF', useExisting: CkColumnDef},
+  ],
+})
+export class CkColumnDef extends CdkColumnDef {
+  /** Unique name for this column. */
```

Comment:
Doc comments are inherited from the base class, no need to duplicate them

Reply:
Done